src/library/dao/autodjcratesdao.cpp

```diff
@@ -138,35 +136,33 @@ void AutoDJCratesDAO::createAndConnectAutoDjCratesDatabase() {
         return;
     }
 
-    // Fill out the first three columns.
-    // Supply default values for the last two.
-    // INSERT INTO temp_autodj_crates (
-    //     track_id, craterefs, timesplayed, autodjrefs, lastplayed)
-    // SELECT crate_tracks.track_id, COUNT (*), library.timesplayed, 0, ""
-    // FROM crate_tracks, library
-    // WHERE crate_tracks.crate_id IN (
-    //     SELECT id
-    //     FROM crates
-    //     WHERE autodj = 1)
-    // AND crate_tracks.track_id = library.id
-    // AND library.mixxx_deleted = 0
-    // GROUP BY crate_tracks.track_id, library.timesplayed;
```

Comment:
We had this comment to have a chance to instantly see the result if the string magic below. 
I would prefer to keep it. This also helps to find the query with find in files in case of an error message. 
I think we can even give more insights what we do here and why.

Reply:
I disagree. I will not write the query twice, both as code and as a comment. The actual query is logged on errors.

Reply:
It was useful in the past since the code below is not readable. Otherwise you have no chance to understand what happens here in the class. It is already hard to understand and now you have removed one of the few hooks. 
At least we need a decent explanation what the query acutally does. 
Since this is probably more work, it is OK for me if you only restore the result of the string magic below. 
Just print it in as qDebug() and copy it here.

Reply:
No, I refuse to add redundancy. I have added a verbose debug log that could be enabled on demand by anyone who needs to inspect the actual query.

Reply:
The debug log contains the explanation of what the query does.